<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderSettingsForm</class>
 <widget class="QWidget" name="SliderSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="disabledLabel">
      <property name="text">
       <string>Disabled</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="disabledCheckBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="valueLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="orientationLabel">
      <property name="text">
       <string>Orientation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="orientationComboBox">
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="invertedLabel">
      <property name="text">
       <string>Inverted</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="invertedCheckBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TODO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
